pkg/app/app_reconcile_test.go

```diff
@@ -52,15 +52,15 @@ func Test_NoInspectReconcile_IfNoDeployAttempted(t *testing.T) {
 		Conditions: []v1alpha1.AppCondition{{
 			Type:    v1alpha1.ReconcileFailed,
 			Status:  corev1.ConditionTrue,
-			Message: "Fetching resources: exit status 1",
+			Message: "Fetching resources: check UsefulErrorMessage in status for more details",
```

Comment:
may be a bit more concise:

`Fetching resources: Error (see .status.usefulErrorMessage for details)`